modules/utils/src/merkle.ts

```diff
@@ -15,7 +16,9 @@ export const generateMerkleTreeData = (transfers: CoreTransferState[]): { root:
   });
 
   // Generate tree
-  const tree = new MerkleTree(leaves, keccak256, { sortPairs: true });
+  const tree = new merkle.Tree();
+  tree.insert_hex_js(leaves);
```

Comment:
This method takes just one leaf at a time. So, you'll need to loop over them. Remember, the intent was that the tree is updated iteratively (though the code you have written is a shorter and less risky path - I haven't profiled to confirm that this will actually be better if we don't keep the tree around. It's worth verifying that this is an improvement over the old code since this is not what was profiled when reporting the improvements.)